requirements.txt

```diff
@@ -5,3 +5,5 @@ uvicorn
 boto3
 flake8
 pandas
+nbconvert
```

Comment:
this is good for getting the tests to pass in CI but we need to add the same requirements to the setup.py file as well

Reply:
looks like you added it to setup.py